client/src/app/site/interaction/components/action-bar/action-bar.component.html

```diff
@@ -54,7 +54,7 @@
         matTooltip="{{ 'Give applause' | translate }}"
         [disabled]="sendsApplause | async"
         [matBadge]="applauseLevelObservable | async"
-        [matBadgeHidden]="!(applauseLevelObservable | async)"
+        [matBadgeHidden]="!(showApplauseLevel && (applauseLevelObservable | async))"
```

Comment:
@tsiegleauq can you please check if this code line is correct - it works. 
If the config value for showApplauseLevel was changed a reload in client is required. Maybe you can fix this?

Reply:
almost. It's usually cleaner to combine Observables in the service to archive that (which is syntactically hard)